hermeto/core/models/input.py

```diff
@@ -54,7 +54,7 @@ def show_error(error: "ErrorDict") -> str:
 
 
 # Supported package managers
-PackageManagerType = Literal["bundler", "cargo", "generic", "gomod", "npm", "pip", "rpm", "yarn"]
+PackageManagerType = Literal["bundler", "cargo", "generic", "gomod", "maven", "npm", "pip", "rpm", "yarn"]
```

Comment:
I don't believe we'll merge this in one go (I think it's been a loong time since we did, probably just at the beginning), so this will have to live in under the `dev-package-managers` (or what the structure holding these is called).